cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Gotcha, so we should actually make it ENV_TOKENS.get in cms also, and since the var is set in both lms.env.json and cms.env.json they'll both pick the same queue (and we'll ensure the queue exists on lms when the substitution happens).